Sources/_MatchingEngine/Regex/Parse/LexicalAnalysis.swift

```diff
@@ -1204,6 +1271,26 @@ extension Source {
     return src.canLexNumberedReference()
   }
 
+  /// Whether a group specifier should be lexed as an atom instead of a group.
+  private func canLexGroupLikeAtom() -> Bool {
```

Comment:
Is this "can" or "should"?

Reply:
Ah yeah it's "should" as we bail from `lexGroupStart` if it returns true, good catch, I'll rename it